airbase-policy/src/main/resources/checkstyle/airbase-checks.xml

```diff
@@ -256,6 +256,14 @@
             <property name="ignoreComments" value="true" />
         </module>
 
+        <module name="RegexpSinglelineJava">
+            <!-- Forbid "new Type[]{",  IntelliJ would reformat adding a space -->
+            <!-- Forbid "new Type[]  {",  IntelliJ would reformat removing a space -->
+            <property name="format" value="new\s++\w++(&lt;\?&gt;)?(\[\])++(?! )\s*+\{" />
```

Comment:
Not sure I understand the last part: `(?! )\s*+\{`. How does that handle the following case: `new X[]    {`?

Reply:
it does not, that's a mistake.

Reply:
thanks, fixed!